types/wlr_output_layout.c

```diff
@@ -270,20 +348,6 @@ struct wlr_output *wlr_output_layout_output_at(struct wlr_output_layout *layout,
 	return NULL;
 }
 
-void wlr_output_layout_move(struct wlr_output_layout *layout,
-		struct wlr_output *output, int lx, int ly) {
```

Comment:
don't remove this.

Reply:
`wlr_output_layout_add` can do the same thing (and does so in master), so what would be the reason for keeping this around?

Reply:
It might make sense. Make an issue for it. It's an unrelated breaking public API change so it needs some discussion and visibility.

Reply:
I guess this might also depend on what is decided in #1013.